lib/zclient.c

```diff
@@ -2285,12 +2287,25 @@ struct interface *zebra_interface_link_params_read(struct stream *s,
 		return NULL;
 	}
 
+	if (changed == NULL)
+		return NULL;
+
```

Comment:
Here you could take another choice: don't perform comparison and always set link param values.

Reply:
Updated with the review comment suggestion.

The parameter 'bool *changed' is now optional.